src/infra/flow-system-inmem/src/services/flow/flow_service_inmem.rs

```diff
@@ -179,6 +179,7 @@ impl FlowServiceInMemory {
                         self.enqueue_auto_polling_flow_unconditionally(start_time, &flow_key)
                             .await?;
                     }
+                    FlowConfigurationRule::CompactingRule(_) => (),
```

Comment:
Make an explaining comment that nothing should be done, only manual triggers are expected